client/components/domains/accordion/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react';
```

Comment:
This should be a type import:
```suggestion
import type { ReactNode } from 'react';
```

Reply:
Cool, I didn't know this syntax 🤙